redpipe-engine/src/main/java/net/redpipe/engine/core/Server.java

```diff
@@ -386,6 +390,7 @@ private JsonObject loadFileConfig(JsonObject config)
 	}
 
 	private void setupSwagger(VertxResteasyDeployment deployment) {
+		ModelConverters.getInstance().addConverter(new RxModelConverter());
```

Comment:
If this line is commented, tests don't pass, else, tests are OK

Reply:
Makes sense.